python/python/ert/enkf/node_id.py

```diff
@@ -17,4 +18,4 @@ def __init__(self, report_step, realization_number):
 
 
 CWrapper.registerType("node_id", NodeId)
-
+registerType( "node_id" , NodeId ) 
```

Comment:
is this the new style wrapper?  Not `TYPE_NAME = "node_id" and delete the `import CWrapper` from head?